webrender/src/resource_cache.rs

```diff
@@ -186,6 +186,7 @@ pub struct ResourceCache {
     cached_images: ResourceClassCache<ImageRequest, CachedImageInfo>,
 
     // TODO(pcwalton): Figure out the lifecycle of these.
+    #[cfg_attr(not(feature = "webgl"), allow(dead_code))]
```

Comment:
We can remove them indeed, but overall in big structs, I didn't think it is worth to duplicate all the constructors just to avoid that field. I can do it if you prefer it that way, I just thought adding fields would become a bit more annoying.